docs/develop.zh-CN.md

```diff
@@ -19,6 +19,14 @@
 
 # Dashboard 开发
 
+## 前置条件
+
+```
+$ git clone -b v2.0 https://github.com/apache/apisix-dashboard.git
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/98618936-296e3e00-233d-11eb-87a0-9c74d5721799.png)

Reply:
the `cd apisix-dashboard` move to the line below